kolibri/core/assets/src/views/KSelect/KeenUiSelect.vue

```diff
@@ -242,7 +254,10 @@
         type: String,
         default: 'Search',
       },
-      filter: Function,
+      filter: {
+        type: Function,
+        default: () => null,
```

Comment:
```suggestion
        default: null,
```

See line 341 of this component. This might cause a bug if you provide a default implementation of `filter` as a function.